### DISCUSSION TITLE
Alignment of Trackname in TCP and the Clip-handle drag-bars

### DISCUSSION THREAD

**Discussion by petersampsonaudacity** [audacity/audacity#1473] [Development]
**Describe the bug**
The trackname in Track Control Panel (TCP) is not aligned with the Clip-handle drag-bars (AKA "affordances")

**To Reproduce**
1) Get some audio
2) Observe the mis-alignment

**Expected behavior**
Either:
a) the height of the Clip-handle drag-bar to be reduced to the same size as the Track Name field in the TCP, or
b) the height of the Track Name field in the TCP to be enlarged to be the same height as the Clip-handle drag-bar

**Screenshots**
Image magnified  x 2.5
![image](https://user-images.githubusercontent.com/50205984/129359126-833506c3-7d4c-445c-9163-add5daa177c3.png)

**Additional information (please complete the following information):**
 - Windows 10 - but assume all OS
 - Audacity 3.1.0/3.0.4

**Additional context**
It would just look much neater and more visually/aesthetically pleasing if the two were in alignment.

**Comment by Paul-Licameli** [comment]
The drag bar is now 20 pixels high.  The close and menu button are 18 pixels high.

The constants in the code that define these are kAffordancesAreaHeight and  kTrackInfoBtnSize.

It should be easy to change those constants to be the same.  The question is which.

I don't know where @jounih stipulated 20 exactly but if that exact number isn't important, reducing it to 18 preserves appearance of all the older things in the interface that were not intended to change, makes drag bars consistent with them, and in the case where a stereo track is minimized, leaves sufficient height to display both drag bars completely, although then the wave data disappear completely.

There are also other open issues, not to show the drag bars for the right channel in the usual cases.

**Reply by jounih** [reply]
18px for the clip handle bars sounds fine and looks like it would fix this. @Paul-Licameli  or @vsverchinsky could you please update kAffordancesAreaHeight to 18px? And please send a link to the build to me/Martin once done, so we can check visually. Thanks

**Comment by petersampsonaudacity** [comment]
@Paul-Licameli @vsverchinsky  any chance that one of you could make this simple change as @jounih requested so that we can test it out please ?

**Comment by petersampsonaudacity** [comment]
This looks better now testing on W10 with audacity-win-3.0.4-alpha-20210817+7949a0d-64bit

Scaled  2.5x
![image](https://user-images.githubusercontent.com/50205984/129868265-35771b6d-a108-4fef-af39-a1c5576b9b1a.png)
